packages/core/src/Stores/client-store.js

```diff
@@ -1965,8 +1969,19 @@ export default class ClientStore extends BaseStore {
         }
 
         const is_client_logging_in = login_new_user ? login_new_user.token1 : obj_params.token1;
+
         if (is_client_logging_in) {
-            window.history.replaceState({}, document.title, sessionStorage.getItem('redirect_url'));
+            const is_pre_appstore = window.localStorage.getItem('is_pre_appstore');
```

Comment:
Is BE gonna add a flag to let us know if a user opted for pre-appstore?

Reply:
With current approach, if the user has opted for pre-appstore and logged in with a different device next time, pre-appstore won't be enabled for him.

Reply:
Its already developed, but not yet deployed. Expected next week Friday